tests/phpunit/WC_Stripe_Account_Test.php

```
@@ -502,22 +503,84 @@ public function test_reconfigure_webhooks_on_update_with_outdated_webhooks() {
+		$this->account->maybe_reconfigure_webhooks_on_update();
+	}
+
+	/**
+	 * Tests that webhook reconfiguration is triggered when the agentic flag
+	 * causes the desired API version to differ from the existing webhook.
+	 */
+	public function test_reconfigure_webhooks_on_update_with_agentic_flag_enabled() {
+		add_filter( 'wc_stripe_is_agentic_commerce_enabled', '__return_true' );
+
+		// Mock a webhook that has current events but the non-agentic API version.
+		$outdated_webhook = (object) [
```

PHPMD (2.15.0)

[error] The variable $outdated_webhook is not named in camelCase. (undefined)

(CamelCaseVariableName)